### ISSUE TITLE
Support list view for Albums, Folders, Calendar, Moments, Labels, Library/Originals

### ISSUE THREAD

**Issue by jimmcslim** [photoprism/photoprism#382]
Would be great if the Albums, Folders, Calendar, Moments, Labels, Library/Originals views allowed these objects to be viewed in list mode - where at the moment only mosaic/card is supported?

**Comment by lastzero** [comment]
We can easily add this later, I don't want to risk introducing new bugs shortly before our release.

**Comment by jimmcslim** [comment]
Absolutely! Have just imported approximately 19,000 photos into a test instance and contributing some UI feedback for some consideration (and quite happy to reindex again if the DB is changing with the release!)

**Comment by danielo515** [comment]
When doing this, you can also keep #583 in mind? I don't think I will ever want to change how I display the folders, specially I don't want them to have the same view as my gallery and vice/versa.
